src/Common/Internal/Filters/AuthenticationFilter.php

```diff
@@ -81,12 +82,12 @@ public function handleRequest(IHttpClient $request)
     /**
      * Does nothing with the response.
      *
-     * @param IHttpClient             $request  HTTP channel object.
-     * @param \HTTP_Request2_Response $response HTTP response object.
+     * @param IHttpClient       $request  HTTP channel object.
+     * @param ResponseInterface $response HTTP response object.
```

Comment:
ResponseInterface is defined in PSR 7 http://www.php-fig.org/psr/psr-7/ and it has all functionality that we need. I don't see a point to create a wrap on top of it. But we may wrap in the future if we have a good reason for it.